assets/base.css

```diff
@@ -3120,3 +3120,41 @@ details-disclosure > details {
 .rte blockquote > * {
   margin: -0.5rem 0 -0.5rem 0;
 }
+
+/* Animations */
+
+.scroll-animation__trigger {
```

Comment:
Let's align on a BEM block name for animations if we're going to have global ones. I think in Kjell's [other ambient motion PR](https://github.com/Shopify/dawn/pull/2342/files#diff-00b03bd62b155daa89082ba52094c182e46422a9e28cd3ec601b3a2291b52718R3126) he used `animate` as the block and `ambient` as the modifier, e.g. `animate--ambient`. Let's be consistent between the two.

I had originally thought the scroll trigger JS and the animation CSS should be separate. In other words, the scroll trigger class does not have any styling on its own, and the styling is in another class. So the rule here would be something like:
```css
.scroll-trigger--active.animate--slide-in {
```

I still think this is the best approach since it will let us have different animations depending on the context. But if you want to keep it simple for now you could just rename this `animate__slide-in` and `animate__slide-in--active` and modify `animations.js` to match.

Reply:
+1 to use separate classes for animation vs trigger. I was writing a similar suggestion.

The `.scroll-animation__trigger--active` name doesn't suggest that it's a slideIn animation and from the JS code, its purpose is just to be an indicator if the element has been in the viewport or not. I would opt for `.scroll-trigger--active.animate--slide-in` even if we don't really need other animation types right now as it keeps the `animations.js` as generic as it was designed to be.